### DISCUSSION TITLE
Case design

### DISCUSSION THREAD

**Discussion by mebs** [mebs/CM4-NAS#6] [Ideas]
Along with the circuit design, I have a case design that is in progress. It's meant to be 3D printed and holds four 3.5" hard drives, a 92mm fan on the bottom and the board with its PCIe daughter.
Here are a few screenshots (EDIT: they're out of date, but still give a good idea):

![image](https://user-images.githubusercontent.com/2614134/102339731-98a20800-3f63-11eb-9f09-5d9f6f422ef1.png)

The idea behind the base design is to help channel the air for the fan, but I mostly think it looks cool and have no idea how much it actually helps.
![image](https://user-images.githubusercontent.com/2614134/102339962-ea4a9280-3f63-11eb-8c6f-3a7dbe5c26ed.png)

When closed.
![image](https://user-images.githubusercontent.com/2614134/102340018-00585300-3f64-11eb-8a4b-bc4437c0d80b.png)

It's meant to expect a 12V barrel jack and an Ethernet connection. A nice improvement would be to power it with POE.
![image](https://user-images.githubusercontent.com/2614134/102340140-27af2000-3f64-11eb-8abe-6a537d3eec88.png)

Section view.
![image](https://user-images.githubusercontent.com/2614134/102340262-5200dd80-3f64-11eb-9712-ba066f56ee6a.png)

**Comment by burra** [comment]
How is it intended to get power to the 3.5" disks?

**Reply by olvint** [reply]
There is a SATA power connector on the board.

**Reply by mebs** [reply]
Yes, with something like this (but replacing the Molex connector with a JST one to plug to the board): 
![image](https://user-images.githubusercontent.com/2614134/103185729-a58bf900-488b-11eb-9ca4-2dfe09ed3111.png)

**Comment by mebs** [comment]
Here's how the case ended up looking like:

![PXL_20210407_182958078](https://user-images.githubusercontent.com/2614134/113934711-16647a80-97c4-11eb-97a2-b27490795be2.jpg)
